subsys/infinispan/src/main/java/org/commonjava/indy/subsys/infinispan/CacheProducer.java

```diff
@@ -179,17 +180,13 @@ private void startCacheManager( String configFile, Boolean isCluster )
                 try
                 {
                     logger.info( "Using CLASSPATH resource Infinispan configuration:\n\n{}\n\n", resourceStr );
+                    mgr = new DefaultCacheManager(
```

Comment:
This looks like it will discard any pre-existing CacheManager instance that gets passed in...is that the intention here?

Reply:
From the code before change, seems this logic is the initialization of the cacheManager. 
But of course we should use some protection here. I think we need a null check here.